README.md

```diff
@@ -48,4 +57,4 @@ In case any player wants to switch characters between rounds, the winner player
 - Delfino Plaza
 - Warioware
 - Halberd (Brawl)
-- Skyloft
\ No newline at end of file
+- Skyloft
```

Comment:
Are you going to remove Skyloft?... Or you want to leave it there?

Reply:
Remove.